### DISCUSSION TITLE
Allow switching between British and American spellings instead of removing words

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#2024] [Ideas] [answered]
Commit [ea1698e](https://github.com/Miodec/monkeytype/commit/ea1698e0098a6d6114fac95078dc062760661d74) removes words with alternate spellings instead of allowing for switching between alternate spellings.

**Comment by Miodec** [comment, marked as answer]
Already a thing - look for british english in the settings.

**Reply by ghost** [reply]
> Already a thing - look for british english in the settings.

Yes but you removed the words from English 5k so they won't show up any more even with British English selected.

**Reply by Miodec** [reply]
> > Already a thing - look for british english in the settings.
> 
> 
> 
> Yes but you removed the words from English 5k so they won't show up any more even with British English selected.

British english doesnt "unlock" or "add" words - it replaces them based on word pairs.

**Reply by ghost** [reply]
Does [removing the entries](https://github.com/Miodec/monkeytype/commit/ea1698e0098a6d6114fac95078dc062760661d74) not remove the word pairs?

**Reply by Miodec** [reply]
> Does [removing the entries](https://github.com/Miodec/monkeytype/commit/ea1698e0098a6d6114fac95078dc062760661d74) not remove the word pairs?

Actually yeah, it seems that I should've replaced those with their American spelling versions. I'll do it today.
